requirements.txt

```diff
@@ -3,6 +3,7 @@ scout-apm==2.17.0
 sentry-sdk==0.19.5
 
 # Production dependencies
+anyascii==0.2.0
```

Comment:
We do not need to have it in our requirements as it will be installed from the Wagtail dependencies [list](https://github.com/wagtail/wagtail/blob/8487c67ebb35984f7694537e779583c7d6b92cf4/setup.py#L38).

Reply:
Removed it now, thanks.